packages/govern-core/contracts/pipelines/GovernQueue.sol

```diff
@@ -244,7 +252,7 @@ contract GovernQueue is IERC3000, AdaptativeERC165, IArbitrable, ACL {
         return _execute(_container.payload, containerHash);
     }
 
-    function settleRejection(ERC3000Data.Container memory _container) public {
+    function settleRejection(ERC3000Data.Container memory _container) internal returns (bytes32, bytes[] memory){
```

Comment:
Since this is an internal function, could we not return anything here and handle the empty returns in the caller?

I don't think we should be trying to make type of thing look "nice" in Solidity :).